dataproc_jupyter_plugin/handlers.py

```diff
@@ -203,7 +203,7 @@ def get(self):
         cloudkms_url = gcp_service_url('cloudkms')
         cloudresourcemanager_url = gcp_service_url('cloudresourcemanager')
         datacatalog_url = gcp_service_url('datacatalog')
-        storage_url = gcp_service_url('storage', default_url='https://storage.googleapis.com/storage/v1/')
+        storage_url = gcp_service_url('storage', default_url='https://content-storage.googleapis.com')
```

Comment:
This should change back to the previous version. In particular, the value returned for the `api_endpoint_overrides/storage` setting should always include the `/storage/<VERSION/` path.